src/components/Modals/Onboarding/Onboarding.vue

```diff
@@ -172,6 +188,24 @@ export default defineComponent({
         (this.onboarding.grad ? getGradFullName(this.onboarding.grad) === '' : false)
       );
     },
+    /**
+     * Display error if the entrance and graduation year are not blank and the graduation semester comes before entrance semester, comparing the season if not blank
+     *
+     * @returns true if graduation semesters comes before entrance semester, false otherwise
+     */
+    isInvalidGraduationSemester(): boolean {
+      const { gradYear } = this.onboarding;
```

Comment:
Maybe I don't know the syntax for this, but is this just taking out the gradYear and entranceYear fields from onboarding?

Reply:
Otherwise, the function makes sense to me

Reply:
yup! our ESLint is set to prefer object destructuring over `this.onboarding.gradYear`